client/src/editorWebviewScript.ts

```diff
@@ -82,6 +88,11 @@ const machine = createMachine<WebViewMachineContext, EditorWebviewScriptEvent>(
       index: 0,
       layoutString: undefined,
       token: undefined,
+      implementations: {
```

Comment:
q: any special reason why delays are not here?

Reply:
I hadn't thought that inline delays were possible. Farzad mentioned that they are, but I don't see much usage for them. We can add them later:

https://github.com/statelyai/xstate-vscode/issues/71

Reply:
is there any significant cost to supporting them right away though?

Reply:
Just the fact that I'd need to run the changes through this chain of 4 PR's, which I don't want to do.